### PR TITLE
HW9

### PR DESCRIPTION
**Pull request by MasiukBeTech**

### CODE REVIEW

File: lesson_9/index9.html [hunk 1]
```diff
@@ -5,47 +5,45 @@
...
-    <div>
-        <h1 class="line">Iryna Masiuk</h1>
-        <form action="C:/всяке/Robot_Dreams/lesson_6/index6.html">
-            <input type="submit" value="Offer job" name="offer_job">
-        </form>
-        <br>
-        <img src="myphoto.jpg" width="240" height="320">
-        <h3 class="line">Junior frontend developer</h3>
+    <div class="top-block">
+        <div class="text-aline">
```
Comment (RomanIlkiv) [inline lesson_9/index9.html]: I don't see you use such a class in your .css
why it's needed?

File: lesson_9/index9.html [hunk 2]
```diff
@@ -5,47 +5,45 @@
...
+            <h1 class="line">Iryna Masiuk</h1>
+            <h3 class="line">Junior frontend developer</h3>
+        </div>
+        <img src="myphoto.jpg">
```
Comment (RomanIlkiv) [inline lesson_9/index9.html]: add `alt` attribute

File: lesson_9/styless9.css [hunk 3]
```diff
@@ -34,14 +22,49 @@ a:hover {
...
+    align-items: center;
+    float: left;
+    clear: left;
+}
+
+
+.skills {
+    display: flex;
+    width: 100% ;
+    justify-content:right;
```
Comment (RomanIlkiv) [inline lesson_9/styless9.css:63]: space after :

File: lesson_9/index9.html (hunk 1 above)
Comment (MasiukBeTech) [reply lesson_9/index9.html]: I forgot to change the name of the class. I fixed

File: lesson_9/index9.html (hunk 2 above)
Comment (MasiukBeTech) [reply lesson_9/index9.html]: added and fixed

File: lesson_9/styless9.css (hunk 3 above)
Comment (MasiukBeTech) [reply lesson_9/styless9.css:63]: my bad. fixed

**Pull request merged**